agents-api/tests/test_docs_queries.py

```diff
@@ -289,7 +295,7 @@ async def _(dsn=pg_dsn, agent=test_agent, developer=test_developer):
     assert result[0].metadata is not None
 
 
-@skip("hybrid search: test container not vectorizing")
+@skip("embedding search: test container not vectorizing")
```

Comment:
hybrid or embedding search here?

Reply:
changed it coz it's the embedding part of it that isn't working anymore